components/shared/constellation/structured_data/transferable.rs

```diff
@@ -185,3 +187,10 @@ impl MessagePortImpl {
         self.state = MessagePortState::Detached;
     }
 }
+
+#[derive(Debug, Deserialize, MallocSizeOf, Serialize)]
+/// A transferable version of the OffscreenCanvas interface.
```

Comment:
I think it would be more accurate to write: "A struct supporting the transfer of `OffscreenCanvas`, which loosely corresponds to the `dataHolder` in <https://html.spec.whatwg.org/multipage/canvas.html#the-offscreencanvas-interface%3Aoffscreencanvas-16>."